packages/flutter_app_badger/pubspec.yaml

```
@@ -0,0 +1,26 @@
+name: flutter_app_badger_tizen
+description: Tizen implementation of the flutter_app_badger plugin
+homepage: https://github.com/flutter-tizen/plugins
+repository: https://github.com/flutter-tizen/plugins/tree/master/packages/flutter_app_badger_tizen
+version: 0.1.0
+
+flutter:
+  plugin:
+    platforms:
+      tizen:
+        pluginClass: FlutterAppBadgerTizenPlugin
+        fileName: flutter_app_badger_tizen_plugin.h
+
+dependencies:
+  flutter:
+    sdk: flutter
+  plugin_platform_interface: ^2.0.2
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  flutter_lints: ^2.0.0
+
+environment:
+  sdk: '>=2.18.0 <3.0.0'
+  flutter: ">=2.5.0"
```

Please add new line.

```suggestion
  flutter: ">=2.5.0"

```